src/services/document-service.js

```diff
@@ -372,6 +402,7 @@ class DocumentService {
       tags: doc.tags || [],
       archived: doc.archived || false,
       origin: doc.origin || DOCUMENT_ORIGIN.internal,
+      originUrl: doc.originUrl || '',
```

Comment:
Shouldn't we have a migration for that (setting it to `null` for internal docs/revs)?

Reply:
Yes we should, added.